src/Core/Auth/Models/Data/OrganizationAdminAuthRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Bit.Core.Auth.Entities;
+using Bit.Core.Auth.Enums;
+
+namespace Bit.Core.Auth.Models.Data;
+
+/// <summary>
+/// Represents an <see cref="AuthRequestType.AdminApproval"/> AuthRequest.
+/// Includes additional user and organization information.
+/// </summary>
+public class OrganizationAdminAuthRequest : AuthRequest
+{
+    /// <summary>
+    /// Email address of the requesting user
+    /// </summary>
+    public string Email { get; set; }
+
+    public Guid OrganizationUserId { get; set; }
+
+    public Guid OrganizationId { get; set; }
```

Comment:
I agree with the GitHub warning, we should add the `new` keyword.

Reply:
Ah, we don't actually even need to override it. I removed the `OrganizationId` here as it wasn't adding anything new and was a leftover from before I added the `OrganizationId` to the base `AuthRequest` entity.